src/controllers/hotelController.js

```diff
@@ -0,0 +1,41 @@
+import hotelModel from "../models/hotelModel";
+
+class Hotel {
+  constructor() {
+    //
+  }
+
+  async createHotel(req, res) {
+    try {
+      await hotelModel.create({
+        name: req.body.name,
+        address: req.body.address,
+        rooms: req.body.rooms,
+      });
+      res.status(200).json({ success: "Hotel created..." });
+    } catch (error) {
+      res.status(500).json({ message: error.message });
```

Comment:
you have not structured the response as mentioned in the previous review. Please, go through the feedback from that review again.